src/cache/cache.rs

```diff
@@ -91,11 +91,9 @@ impl<T: Read + Seek + Send> ReadSeek for T {}
 
 /// Data stored in the compiler cache.
 pub struct CacheRead {
-    zip: ZipArchive<Box<dyn ReadSeek>>,
+    zip: ZipArchive<Box<dyn ReadSeek + Send>>,
```

Comment:
`pub trait ReadSeek: Read + Seek + Send {}` is implicitly `Send` by definition. This might be the rustc issue that causes some discomfort: https://github.com/rust-lang/rust/issues/78700

Reply:
Oh, good catch! I added that explicitly here so that we don't need an `unsafe impl Send for CacheRead`; I'd prefer leaving figuring these auto traits to rustc whenever possible.

Should I add a FIXME pointing to https://github.com/rust-lang/rust/issues/78700?

Reply:
It seems adding the extra bound is unnecessary as per https://github.com/paritytech/sccache/pull/40#issuecomment-811288004, so I removed this change